tests/hw-accel/kmm/modules/tests/signing-test.go

```diff
@@ -217,19 +218,21 @@ var _ = Describe("KMM", Ordered, Label(kmmparams.LabelSuite, kmmparams.LabelSani
 				Create()
 			Expect(err).ToNot(HaveOccurred(), "error while creating preflight")
 
-			By("Await build pod to complete build")
```

Comment:
And same here. Consecutive runs leaves this images in place, which cause this issue